webview/src/setup/components/dvc/Dvc.tsx

```diff
@@ -31,26 +31,42 @@ export type DvcProps = {
 export const Dvc: React.FC<DvcProps> = ({
   canGitInitialize,
   cliCompatible,
+  dvcCliDetails,
   isPythonExtensionInstalled,
   needsGitInitialized,
   projectInitialized,
   pythonBinPath,
   setSectionCollapsed,
   isExperimentsAvailable
 }) => {
+  const children = (
+    <>
```

Comment:
If you use `PropsWithChildren` inside your other component, you can remove the fragment here as children will be nullable.